src/Engines/TestWare.Engines.Appium/Factory/AppiumDriverFactory.cs

```diff
@@ -1,24 +1,56 @@
-﻿using OpenQA.Selenium.Appium;
+﻿using OpenQA.Selenium.Appium;
 using TestWare.Engines.Appium.Configuration;
 
 namespace TestWare.Engines.Appium.Factory;
 
 internal static class AppiumDriverFactory
 {
     public static IAppiumDriver Create(Capabilities capabilities)
+    {
+
+        IAppiumDriver result = capabilities.GetPlatform() switch
+        {
+            SupportedPlatforms.Android => CreateAndroidDriver(capabilities),
+            SupportedPlatforms.IOS => CreateIOSDriver(capabilities),
+            SupportedPlatforms.Invalid => throw new NotImplementedException(),
+            _ => throw new NotSupportedException($"Browser type is invalid."),

```

Comment:
Error message should be "Platform type is invalid"

Reply:
done